help/start-a-call.md

```diff
@@ -4,16 +4,38 @@
 
 {!start-composing.md!}
 
-1. Click the **video camera** (<i class="fa fa-video-camera"></i>) at the bottom of
-the compose box. This will insert a **Join video call.** link into your message.
+{tab|desktop-web}
+
+1. Click the **video camera** (<i class="fa fa-video-camera"></i>) icon at the
+   bottom of the compose box. This will insert a **Join video call.** link into
+   your message.
 
 1. Send the message.
 
 1. Click on the link in the message to start or join the call.
 
 !!! tip ""
 
-    You can replace the "Join video call." label for the link with any text you like.
+    You can replace the "Join video call." label for the link with any text you
+    like.
+
+{tab|mobile}
+
+1. Navigate to a stream, topic, or direct message view.
```

Comment:
Hmm... in the stream view, the icon is there, but to send the message you're prompted to add a topic. I don't know if we should note that or not somewhere in the instructions?

Reply:
Yeah, I think we should since that's what we did in http://zulip.com/help/quote-and-reply#forward-a-message